xicam/spectral/ingestors/NXcxi_ptycho.py

```diff
@@ -66,6 +66,7 @@ def ingest_cxi(paths):
     sorted_entry_list = sorted(h5_entry_dict.items(), key=lambda item: item[1])
     # TODO check if energies are "continuous" or have interruptions
     # the removal of last for entries is only temporary due to the example dataset
+    # FIXME what to do if some energies are 'out of range'?
```

Comment:
refer to nx tycho to see if we can put these reference energies in a different field (may have to extend the spec)

Reply:
heuristics, or acquisition-side definition of the reference energies
(just writing these comments for later notes - fine to keep this in for now)